Source/wi_stuff.c

```diff
@@ -1945,7 +1945,7 @@ void WI_DrawBackground(void)
   else if (gamemode == commercial || wbs->epsd >= 3)
     strcpy(name, "INTERPIC");
   else 
-    sprintf(name, "WIMAP%d", wbs->epsd);
+    sprintf(name, "WIMAP%1.1d", wbs->epsd);
```

Comment:
snprintf?

Reply:
Wow, snprintf doesn't work. Is this ok: `sprintf(name, "WIMAP%hhu", (byte) wbs->epsd);`?